src/pages/index.jsx

```diff
@@ -272,6 +277,17 @@ const QuickstartsPage = ({ data, location }) => {
                 </Button>
               ))}
             </FormControl>
+            <FormControl>
+              <Label htmlFor="quickstartCategory">Can build QuickStart</Label>
+                <Button
+                  type="button"
+                  key={RESERVED_QUICKSTART_IDS.BUILD_YOUR_OWN_QUICKSTART}
+                  onClick={() => buildYourOwn(QUICKSTARTS_REPO)}
+                  variant={Button.VARIANT.PRIMARY}
+                >
+                  Build your own
+                </Button>
```

Comment:
Our theme actually provides a great way to handle linking and opening in a new tab via the button component. 
The `as` prop allows you to turn into into an external link, docs [here](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#button). **Note**: You'll need to import `ExternalLink` from the `@newrelic/gatsby-theme-newrelic` package.
So, we can do something like this:
```suggestion
                <Button
                  type="button"
                  key={RESERVED_QUICKSTART_IDS.BUILD_YOUR_OWN_QUICKSTART}
                  variant={Button.VARIANT.PRIMARY}
                  as={ExternalLink}
                  to={QUICKSTARTS_REPO}
                  instrumentation={{
                    component: 'SideNav',
                    eventName: 'instantObservability',
                    category: 'BuildYourOwnQuickstartClick',
                  }}
                >
                  Build your own
                </Button>
```
This allows you to avoid having to `window.open` and keeps the link accessible. I also added in the `instrumentation` prop, that helps our analytics tracking identify events.

Reply:
Thank you very much @aswanson-nr for your suggestions.
After discussing with the team, I was told to close this PR as there is no such requirement now.
Hence I am closing this PR.